docs/security/access-control.md

```diff
@@ -90,6 +90,18 @@ The owner of each entity has implicit administrative class privilege, for exampl
 Only the Owner of a securable object can fully manage that resource.
 If a securable object needs to be managed by more than one person at the same time, the owner is assigned to a user group.
 
+The metadata object which supports ownership is as follows:
```

Comment:
"that supports..."

Reply:
`which` or `that` is ok. In America, more people prefer using `that`.

Reply:
It really should be "The metadata objects that support ownership are:". You could possibly leave in "are as follows" but it's unnecessarily verbose.